mkdocs.yml

```diff
@@ -10,6 +10,7 @@ nav:
   - About OpenSAFELY:
       - Our philosophy: open-methods.md
       - Security: security-levels.md
+      - Five Safes: five-safes.md
```

Comment:
```suggestion
      - Safe research access to data: five-safes.md
```

Reply:
Good idea. I've updated this to `Safe data access: The Five Safes`, so it's best of both